remoteit/desktop
# [DESK-784] manage device lists menu

### Test plan



1. Sign in
2. Go to device list of account with linked lists
3. Click the device lists menu
4. You should see a manage lists menu item at the bottom.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-784>
